### ISSUE TITLE
The Sphereon SSI agent is not working with Docker Compose because it does not support OID4VCI draft 13.

### ISSUE THREAD

**Issue by Taki-Kazuhiro** [Sphereon-Opensource/did-siop-auth-web-demo#110]
Hi i'm trying to setup the 3 docker container, but i have a problem with the ssi agent.
I launch the .sh script (./install-configs.sh sphereon http://<my_local_ip>:5000) to setup the conf files, but when i launch the container it exits with the following error:
'IssuerMetadata should be from type v1_0_13 or higher.'

I suspect this is because the demo is not yet compatible with OID4VCI draft 13. When will the demo be updated to reflect this?

Thanks in advance

**Comment by nklomp** [comment]
Yeah sorry for that. Currently the SDK is in a bit of a rough/unstable phase. In the end it is for the better as it improves some abstraction, removes requirements on assuming we are always operating with DIDs and improves consistency. In the short term it means some of our examples are out of sync. Of course we will fix them as soon as the dust settles a bit

**Comment by Taki-Kazuhiro** [comment]
Understood. Thank you for the update, and I look forward to the stable release of the SDK.
